packages/ui/src/domainModules/collectionMammals/components/MammalManager/MainColumn/RecordForm/FormRow/FormSectionNavigation/index.js

```diff
@@ -40,25 +46,84 @@ const propTypes = {
   onSetActiveFormSection: PropTypes.func.isRequired,
   onShowAllFormSections: PropTypes.func.isRequired,
   showAllFormSections: PropTypes.bool.isRequired,
+  // taxonName: PropTypes.shape({
+  //   attributes: PropTypes.shape({
+  //     name: PropTypes.string,
+  //   }),
+  // }),
 }
 const defaultProps = {
   activeFormSectionIndex: undefined,
   catalogNumber: undefined,
+  // taxonName: undefined,
 }
 
 class FormSectionNavigation extends PureComponent {
-  render() {
+  renderSection(index, name) {
     const {
       activeFormSectionIndex,
+      onSetActiveFormSection: handleSetActiveFormSection,
+    } = this.props
+
+    const isActive = index === activeFormSectionIndex
+
+    if (isActive) {
```

Comment:
it's unnecessary to duplicate all this markup. you can simply set the active prop like this
```
        <Step
          active={index === activeFormSectionIndex}
          key={index}
          onClick={event => handleSetActiveFormSection(event, index)}
        >
```